game-core/src/main/java/games/strategy/triplea/attachments/PlayerAttachment.java

```diff
@@ -535,6 +552,12 @@ public void validate(final GameData data) {}
                 this::setGiveUnitControl,
                 this::getGiveUnitControl,
                 this::resetGiveUnitControl))
+        .put("giveUnitControlInAllTerritories",
+            MutableProperty.of(
+                this::setGiveUnitControlInAllTerritories,
+                this::setGiveUnitControlInAllTerritories,
+                this::getGiveUnitControlInAllTerritories,
+                this::resetGiveUnitControlInAllTerritories))
```

Comment:
I'm not 100% sure anymore, but I think there's a method that allows to provide a simple mapping function (getBool in this case) instead of another setter for string values.

Reply:
Not that I'm aware of. I don't see any other boolean properties using an alternative.

Reply:
In this case it's probably just an idea I had at some point

Reply:
Nevermind, I found it:
https://github.com/triplea-game/triplea/blob/fd68e4850b0c1e2519ff49cd0219b8e14c5fe18a/game-core/src/main/java/games/strategy/engine/data/MutableProperty.java#L156-L170

This would reduce the amount of boilerplate code slightly

Reply:
Updated.